src/core/event/EventManager.js

```diff
@@ -0,0 +1,60 @@
+/* global console */
+
+export default class EventManager {
+  constructor(nextHandlers) {
+    this._nextHandlers = nextHandlers;
+  }
+
+  attach(element) {
+    for (const eventType of this.constructor.handledRawEventTypes) {
+      element.addEventListener(eventType, this[eventType]);
```

Comment:
Another thought: If we add handlers, should we also have a way of removing them? Maybe a TODO for the design?